### PR TITLE
remove trigger for all time call signature

### PR DESCRIPTION
**Pull request by BCecatto**
### Why this PR?
- Remove some trigger to call every action in story to call signature.
### Where review?
 - I believe that all is correct, but is good see if upload of files don't broken is some way.

### CODE REVIEW

File: app/assets/javascripts/views/project_view.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import AttachmentOptions from 'models/attachmentOptions'
+import StoryAttachment from 'components/story/StoryAttachment';
 var StoryView = require('./story_view');
 var IterationView = require('./iteration_view');
 var ColumnView = require('./column_view');
 var ColumnVisibilityButtonView = require('./column_visibility_button_view');
 var HistoryView = require('./history_view');
-
```
Comment (andrerpbts) [inline app/assets/javascripts/views/project_view.js]: I think this is a good space, can we keep it?

Comment (BCecatto) [reply app/assets/javascripts/views/project_view.js]: done, and space back.

File: app/assets/javascripts/views/story_view.js [hunk 2]
```diff
@@ -75,20 +75,7 @@ module.exports = FormView.extend({
...
-    const attachmentOptions = new AttachmentOptions({
-      refreshCallback: this.setAttachinaryOptions.bind(this)
-    });
-    attachmentOptions.fetch();
-  },
-
-  setAttachinaryOptions: function(options) {
-    this.attachinaryOptions = options;
-    this.render();
+    this.attachinaryOptions = window.attachinarySignature
```
Comment (talyssonoc) [inline app/assets/javascripts/views/story_view.js]: Will this be refreshed when the signature is fetched again? Is it what happens here https://github.com/Codeminer42/cm42-central/blob/4588c860155d2c6a44b6db49e135f63e7ecfc0d2/app/assets/javascripts/views/story_view.js#L520 ?

Comment (BCecatto) [reply app/assets/javascripts/views/story_view.js]: I think I answer this in chat, sorry I'm late :(

**Pull request merged**